app/src/main/java/org/simple/clinic/home/overdue/OverdueScreen.kt

```diff
@@ -97,8 +116,13 @@ class OverdueScreen(context: Context, attrs: AttributeSet) : RelativeLayout(cont
     }
   }
 
-  fun updateList(overdueAppointments: List<OverdueAppointment>) {
-    overdueListAdapter.submitList(OverdueAppointmentRow.from(overdueAppointments, userClock, dateFormatter))
+  fun updateList(overdueAppointments: List<OverdueAppointment>, showDiagnosisLabel: Boolean) {
```

Comment:
Since showing/hiding the diagnosis label is a presentation detail, can we rename this parameter to `diabetesManagementEnabled` instead?

Reply:
done